src/app/shared/visitor-navbar/style.scss

```diff
@@ -0,0 +1,44 @@
+@use '@angular/material' as mat;
+@use '../../styles/palettes' as palette;
+
+.nav-bar {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  top: 0;
+  position: sticky;
+  background-color: mat.get-color-from-palette(palette.$gray-palette, 100);
+  height: 8vh;
+  transition-duration: 300ms;
+  transform: translateY(0);
+  &--visi {
```

Comment:
co to za skrót `visi` ?

Reply:
od visible skrót, ale faktycznie mało czytelne